pytorch_lightning/strategies/parallel.py

```diff
@@ -85,6 +85,14 @@ def distributed_sampler_kwargs(self):
         distributed_sampler_kwargs = dict(num_replicas=len(self.parallel_devices), rank=self.global_rank)
         return distributed_sampler_kwargs
 
+    @property
+    def parallel_devices(self):
+        return self._parallel_devices
+
+    @parallel_devices.setter
+    def parallel_devices(self, parallel_devices):
```

Comment:
Type hint

Reply:
Will add all the type hints for this file. :)

Reply:
let's not add all type hints for this file, just for the new methods as suggested by kaushik. typing outside of acc connector should be addressed in separate issues, if possible.